#!/usr/bin/env python
PACKAGE = "emergency_stop"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("angle_front",                  int_t, 0, "An Integer parameter", 40,  0, 180)
gen.add("angle_back",                   int_t, 0, "A double parameter", 40,  0, 180)
gen.add("break_distance",               double_t, 0, "A double parameter", 0.45, 0.0, 2.0)
gen.add("break_distance_based_on_speed",bool_t, 0, "A Boolean parameter",  False)
gen.add("reverse_minimum_distance",     double_t, 0, "A double parameter", 0.28, 0.0,  1.0)
gen.add("forward_minimum_distance",     double_t, 0, "A double parameter", 0.07, 0.0,  1.0)

exit(gen.generate(PACKAGE, "emergency_stop", "EmergencyStop"))
